packages/nextcloud/bin/generate_exports.dart

```diff
@@ -5,18 +5,24 @@ import 'package:dynamite/src/helpers/dart_helpers.dart';
 import 'package:path/path.dart' as p;
 
 void main() {
-  final files =
-      Directory('lib/src/api').listSync().cast<File>().where((final file) => file.path.endsWith('.openapi.dart'));
+  final files = Directory('lib/src/api')
+      .listSync()
+      .cast<File>()
+      .where((final file) => file.path.endsWith('.openapi.dart'))
+      .toList()
+    ..sort((final a, final b) => a.path.compareTo(b.path));
 
-  final idStatements = <String>[];
+  final idStatements = StringBuffer();
 
   for (final file in files) {
     final basename = p.basename(file.path);
     final id = basename.substring(0, basename.length - 13);
     final variablePrefix = toDartName(id);
     final classPrefix = toDartName(id, uppercaseFirstCharacter: true);
 
-    idStatements.add("  static const $variablePrefix = '$id';");
+    idStatements
+      ..writeln('  /// ID for the $variablePrefix app.')
```

Comment:
```suggestion
      ..writeln('  /// ID for the $id app.')
```

This should be the ID because the variablePrefix is just a converted version that is not the real name.